kolibri/plugins/coach/assets/src/views/recent-page/channel-list.vue

```diff
@@ -43,10 +81,15 @@
           },
         };
       },
+      lastActiveText(channel) {
+        const trArgs = this.lastActive[channel.id];
+        return this.$tr('timePassed', trArgs);
```

Comment:
This line was the motivation behind the format of the `lastActive` object.

see https://github.com/learningequality/kolibri/pull/1171/files#diff-fea9e3e75d8fe0e1e96da5074cd65528R67

and https://github.com/learningequality/kolibri/pull/1171/files#diff-fea9e3e75d8fe0e1e96da5074cd65528R109